packages/beachball/src/__tests__/changelog/renderChangelog.test.ts

```diff
@@ -0,0 +1,85 @@
+import { MarkdownChangelogRenderOptions, renderChangelog } from '../../changelog/renderChangelog';
+
+const previousContent = `# Change Log - foo
+
+This log was last generated on Wed, 21 Aug 2019 21:20:40 GMT and should not be manually modified.
+
+## 1.2.0
+
+(content here)
+`;
+
+describe('renderChangelog', () => {
+  function getOptions(): MarkdownChangelogRenderOptions {
+    return {
+      isGrouped: false,
+      newVersionChangelog: {
+        date: new Date('Thu Aug 22 2019 14:20:40 GMT-0700 (Pacific Daylight Time)'),
+        name: 'foo',
+        tag: 'foo_v1.2.3',
+        version: '1.2.3',
+        comments: {
+          major: [],
+          minor: [
+            { comment: 'Awesome change', author: 'user1@example.com', commit: 'sha1', package: 'foo' },
+            { comment: 'Boring change', author: 'user2@example.com', commit: 'sha2', package: 'foo' },
+          ],
+          patch: [
+            { comment: 'Fix', author: 'user1@example.com', commit: 'sha3', package: 'foo' },
+            { comment: 'stuff', author: 'user2@example.com', commit: 'sha4', package: 'foo' },
+          ],
+        },
+      },
+      previousContent,
+      previousJson: {} as any,
+      changelogOptions: {},
+    };
+  }
+
+  it('handles no previous content', () => {
+    const options = getOptions();
+    options.previousContent = '';
+    expect(renderChangelog(options)).toMatchSnapshot();
```

Comment:
thanks for adding comprehensive tests! (great idea using snapshot)

we have some tests in changelog.test.ts too in writeChangelog, could you change to snapshot as well?

Reply:
I updated them to only use snapshots. If you want me to also keep any of the explicit tests just let me know and I'll add it back.

Reply:
looks great. thank you!